patches/rt-base-configure.patch

```diff
@@ -0,0 +1,26 @@
+--- a/configure.ac
++++ b/configure.ac
+@@ -42,10 +42,6 @@
+ PKG_CHECK_MODULES([CPPUNIT], [cppunit],, [no_cppunit="yes"])
+ PKG_CHECK_MODULES([DEPENDENCIES], [libtorrent >= 0.13.7])
```

Comment:
I checked that this patch applies successfully against both 0.13.6/0.9.6 and 0.13.7/0.9.7, but I'm happy to change this if you want.